webapp/src/main/webapp/templates/freemarker/page/partials/languageSelector.ftl

```diff
@@ -1,18 +1,12 @@
 <#-- $This file is distributed under the terms of the license in LICENSE$ -->
 
-<#--
-  How can this done with images instead of buttons containing images?
-  Why don't the "alt" values show as tooltips?"
-  What was the right way to do this?
- -->
-
 <#-- This is included by identity.ftl  -->
 <#if selectLocale??>
 <li><ul class="language-dropdown">  <li id="language-menu"><a id="lang-link" href="#" title="user">${i18n().select_a_language}</a><ul class="sub_menu">
     <#list selectLocale.locales as locale>
 
             <li <#if locale.selected>status="selected"</#if>>
-                	<a href="${selectLocale.selectLocaleUrl}?selection=${locale.code}" title="${i18n().select_locale} -- ${locale.label}"><img src="${locale.imageUrl}" title="${i18n().select_locale} -- ${locale.label}" height="15" style="vertical-align:middle" alt="${locale.label}"/></a>
+                	<a href="${selectLocale.selectLocaleUrl}?selection=${locale.code}" title="${i18n().select_locale} -- ${locale.label}">${locale.label}<#if locale.country?has_content> (${locale.country})</#if></a>
```

Comment:
Please also update the comment below that includes mention of `imageUrl` (line 23 in languageSelector.ftl)